training/components.py

```diff
@@ -90,12 +129,13 @@ def pytorchjob_manifest_op(
                   annotations:
                     sidecar.istio.io/inject: 'false'
                 spec:
+                  nodeSelector:
+                    kubernetes.io/hostname: wrk-6
```

Comment:
Can you remove this? its very specific to our cluster

Reply:
I can, but removing this would not force the master and the worker pods to be on the same node. That would give the PVC attach error.